src/components/EmptyState/README.md

```diff
@@ -163,6 +163,29 @@ Use to explain a single feature before merchants have used it.
 </EmptyState>
 ```
 
+### Empty state with footer
```

Comment:
Empty state with subdued footer
Use this component to provide additional but non-critical context for a new product or feature. Can also be used to include a subdued CTA for secondary or tertiary actions.